randovania/data/json_data/prime2.txt

```diff
@@ -10029,6 +10038,7 @@ Asset id: 1089946750
               Any of the following:
                   Scan Visor and Combat/Scan Dash (Intermediate)
                   Morph Ball and Roll Jump (Intermediate)
+          Scan Visor and Extended Dash (Hypermode) and Dark World Damage ≥ 50 and Activate Safe Zone
```

Comment:
Also no SJ?

Reply:
ATS extended dash is fine. I forgot SJ for watch station though >_>